seatunnel-connectors-v2/connector-tdengine/src/main/java/org/apache/seatunnel/connectors/seatunnel/tdengine/exception/TDengineConnectorErrorCode.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.tdengine.exception;
+
+import org.apache.seatunnel.common.exception.SeaTunnelErrorCode;
+
+public enum TDengineConnectorErrorCode implements SeaTunnelErrorCode {
+    SQL_OPERATION_FAILED("TDengine-01", "execute sql failed"),
```

Comment:
SQL_OPERATION_FAILED --> CommonErrorCode.SQL_OPERATION_FAILED
READER_FAILED --> CommonErrorCode.READER_OPERATION_FAILED
WRITER_FAILED --> CommonErrorCode.WRITER_OPERATION_FAILED
TYPE_MAPPER_FAILED --> CommonErrorCode.UNSUPPORTED_DATA_TYPE

CONNECTION_FAILED should be removed and fall into writer or reader operation failed.